### PR TITLE
Add BitMEX exchange API

### PR DESCRIPTION
**Pull request by Darkbladecr**
Added a new plugin for the [BitMEX cryptocurrency exchange](https://www.bitmex.com/app/restAPI).

### CODE REVIEW

**Comment by umitanuki** [comment]
Plugin for each exchange is awesome. Save $49. Also polygon doesn't seem like it supports higher resolution than day

**Comment by blonc** [comment]
i presume if they are all similar tech IE WS or REST then easy enough to do.  poly supports trade/quotes and higher.   as well has historical data back to 2016 in trade form.

**Comment by umitanuki** [comment]
Regardless, we love to embrace diverse use cases and needs as an opensource project.

**Comment by umitanuki** [comment]
ccxt is also coming to Go.  Might be interesting too. https://github.com/ccxt/ccxt/issues/1952

**Comment by blonc** [comment]
Actually that would be awesome... CCXT and marketstore would be a pretty great combination. That’s looks like a great library, jeez, no matter how much you think you know you always run into new library.

**Comment by Darkbladecr** [comment]
@blonc I have recently read up on polygon and it is great that it can aggregate all this data, however I have no interest in paying for a service that I can build myself, which is what I am sharing here. I hope to add more exchanges in the future.

Please do review the code @umitanuki I am brand new to golang from javascript and python. Everything is working on my linux server, but it would be great to have someone else try it!

**Comment by blonc** [comment]
>@blonc I have recently read up on polygon and it is great that it can aggregate all this data, however I have no interest in paying for a service that I can build myself, which is what I am sharing here. I hope to add more exchanges in the future. ...

@Darkbladecr agree for sure, was just making a FYI. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: contrib/bitmexfeeder/api/api.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		"reverse":   []string{"false"},
+		"startTime": []string{from.String()},
+	}
+	uri, err := url.Parse(baseURL + "/api/v1/trade/bucketed")
+	if err != nil {
+		return nil, err
+	}
+	uri.RawQuery = values.Encode()
+	reqURL := uri.String()
+	log.Println(reqURL)
```
Comment (rocketbitz) [inline contrib/bitmexfeeder/api/api.go]: either remove or use glog

**Comment by Darkbladecr** [comment]
@rocketbitz removed

**Comment by rocketbitz** [comment]
Great, thanks @Darkbladecr.

**Pull request merged**